dcm2rpy identifier not found while compiling OVRHeadset

When I try to compile the `OVRHeadset` plugin on Windows I get the following error
```
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(955): error C2039: 'dcm2rpy': is not a member of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\libYARP_math\src\yarp/math/FrameTransform.h(18): note: see declaration of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(955): error C3861: 'dcm2rpy': identifier not found
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(956): error C2039: 'dcm2rpy': is not a member of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\libYARP_math\src\yarp/math/FrameTransform.h(18): note: see declaration of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(956): error C3861: 'dcm2rpy': identifier not found
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(960): error C2039: 'rpy2dcm': is not a member of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\libYARP_math\src\yarp/math/FrameTransform.h(18): note: see declaration of 'yarp::math'
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(960): error C3861: 'rpy2dcm': identifier not found
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(962): error C3861: '*': identifier not found
17>C:\Users\icub\sb\robotology\YARP\src\devices\ovrheadset\OVRHeadset.cpp(963): error C3861: '*': identifier not found
```

**Configuration (please complete the following information):**
 - OS: windows 
 - yarp version: latest master https://github.com/robotology/yarp/commit/f91ceb574b133b0cd45981b959d536dd62ac2e5e
-  compiler: Visual Studio 2019

--- a/src/devices/ovrheadset/OVRHeadset.cpp
+++ b/src/devices/ovrheadset/OVRHeadset.cpp
@@ -36,7 +36,7 @@
 #include <yarp/sig/Image.h>
 #include <yarp/os/Property.h>
 #include <yarp/os/SystemClock.h>
-#include <yarp/math/FrameTransform.h>
+#include <yarp/math/Math.h>
 
 #include <cmath>
 #include <mutex>

